engine/evd.mli

```diff
@@ -77,6 +77,14 @@ sig
 
 end
 
+module Abstraction : sig
+  type t = bool list
```

Comment:
Can we use a variant instead of `bool` so that the meaning of `true` would be clearer?

Reply:
How do "Abstract" and "Transparent" sound?

Reply:
In the comment below you talk about abstraction and imitation, IIRC.

Reply:
Right. Abstract and Imitate are the better names